README.md

```diff
@@ -66,25 +66,23 @@ when processed, produce Markdown content. Here's an example:
 links for the reply type. Note: do not use `@reply-type` specifiers; use only the Markdown link.
 
 ```md
-@simple-string-reply: [Simple string reply](https://redis.io/docs/reference/protocol-spec#simple-strings)
-@simple-error-reply: [Simple error reply](https://redis.io/docs/reference/protocol-spec#simple-errors)
-@integer-reply: [Integer reply](https://redis.io/docs/reference/protocol-spec#integers)
-@bulk-string-reply: [Bulk string reply](https://redis.io/docs/reference/protocol-spec#bulk-strings)
-@array-reply: [Array reply](https://redis.io/docs/reference/protocol-spec#arrays)
-@nil-reply: [Nil reply](https://redis.io/docs/reference/protocol-spec#bulk-strings)
-@null-reply: [Null reply](https://redis.io/docs/reference/protocol-spec#nulls)
-@boolean-reply: [Boolean reply](https://redis.io/docs/reference/protocol-spec#booleans)
-@double-reply: [Double reply](https://redis.io/docs/reference/protocol-spec#doubles)
-@big-number-reply: [Big number reply](https://redis.io/docs/reference/protocol-spec#big-numbers)
-@bulk-error-reply: [Bulk error reply](https://redis.io/docs/reference/protocol-spec#bulk-errors)
-@verbatim-string-reply: [Verbatim string reply](https://redis.io/docs/reference/protocol-spec#verbatim-strings)
-@map-reply: [Map reply](https://redis.io/docs/reference/protocol-spec#maps)
-@set-reply: [Set reply](https://redis.io/docs/reference/protocol-spec#sets)
-@push-reply: [Push reply](https://redis.io/docs/reference/protocol-spec#pushes)
+@simple-string-reply: [Simple string reply](https://placeholderkv.io/docs/reference/protocol-spec#simple-strings)
```

Comment:
Also all `placeholderkv.io` links are non-existent as they are just "placeholders". We will follow up in the future PRs.